templates/concourse-worker-watchdog.j2

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# {{ ansible_managed }}
+
+while(true); do
+    FAIL=0
+
+    curl 127.0.0.1:7777/ping || FAIL=1
```

Comment:
7777 seems to be garden. Would be nice to also have a curl on 127.0.0.1:8888 (concourse worker)